x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -386,12 +416,19 @@ SymbolsAndShapes.displayName = 'SymbolsAndShapes';
  *  3. `<use>` elements can be handled by compositor (faster)
  */
 const SymbolDefinitionsComponent = memo(({ className }: { className?: string }) => {
-  const isDarkMode = useUiSetting<boolean>(DEFAULT_DARK_MODE);
+  const isDarkMode = useUiSetting<boolean>(defaultDarkMode);
+  const resolverComponentInstanceID = useSelector(selectors.resolverComponentInstanceID);
   return (
     <svg className={className}>
       <defs>
```

Comment:
:grey_question: this reflects "1 defs for each resolver". I think we should try instead for "1 defs for all Resolvers".

Reply:
_but_ this is _more_ correct than what we're doing now (1 def per resolver with duplicate IDs) so we should merge this 1st.